dockerunit-core/src/main/java/com/github/qzagarese/dockerunit/ServiceInstance.java

```diff
@@ -119,5 +119,15 @@ public Status getStatus() {
 		return status;
 	}
     
+	/**
+	 * Checks whether the instance is in the specified status.
+	 * 
+	 * 
+	 * @param status the specified status
+	 * @return true if the instance is in the specified status, false otherwise.
+	 */
+	public boolean hasStatus(Status status) {
+	    return getStatus().equals(status);
```

Comment:
readability niptpick, but why not just `this.status.equals(status)`? unless the public accessors have logic, it's usually preferable to just use the member variable as it saves you a "trip" to the accessor method (when reading the code) to see what it's doing.